background/commands.ts

```diff
@@ -1,28 +1,34 @@
-class KFCommands {
 
+const enum Command {
```

Comment:
Can we have this in a separate file (following the same approach as you used for the Actions)